waiter/test/waiter/scheduler/kubernetes_test.clj

```diff
@@ -813,6 +814,10 @@
         (testing "should work with valid configuration"
           (is (instance? KubernetesScheduler (kubernetes-scheduler base-config))))
 
+        (testing "should retain custom plugin options"
+          (is (= (:custom-options base-config)
+                 (-> base-config kubernetes-scheduler :custom-options))))
```

Comment:
> We should also add a unit tests to verify the values received by the k8s scheduler.

Did you want additional tests, or was this test sufficient?

Reply:
This is good.